webxrlayers-1.bs

```diff
@@ -769,21 +771,26 @@ is initialized.
 <pre class="idl">
 dictionary XRProjectionLayerInit {
   XRTextureType textureType = "texture";
+  GLenum colorFormat = 0x1907; // RGBA
+  GLenum depthFormat = 0x84FA; // UNSIGNED_INT_24_8_WEBGL
```

Comment:
Here and below, shouldn't this more accurately be `depthStencilFormat`?

Also, it would be nice if we could use this to eliminate the `depth` and `stencil` booleans in the same way that specifying the `colorFormat` implicitly eliminates the need for `alpha`. It's a bit awkward because it's a `GLenum`, but we could either make it nullable or specify that passing `gl.NONE` (0) indicates that no depth/stencil buffer needs to be provided. What do you think?

Reply:
> Here and below, shouldn't this more accurately be `depthStencilFormat`?

I agree. I will make that change.

> Also, it would be nice if we could use this to eliminate the `depth` and `stencil` booleans in the same way that specifying the `colorFormat` implicitly eliminates the need for `alpha`. It's a bit awkward because it's a `GLenum`, but we could either make it nullable or specify that passing `gl.NONE` (0) indicates that no depth/stencil buffer needs to be provided. What do you think?

I *think* it makes more sense to make it nullable .

Reply:
It certainly feels more "webby". Easier to infer the intent from reading the IDL that way too.